Source/Redemption/Private/MovingPlatform.cpp

```diff
@@ -0,0 +1,110 @@
+#include "MovingPlatform.h"
+#include "Interactable.h"
+
+#define ACCEPTANCE_DISTANCE 5
```

Comment:
This should probably be in the header instead.

Reply:
fair, I will change and merge